### ISSUE TITLE
Incorrect indentation in LambdaRewriter

### ISSUE THREAD

**Issue by svick** [dotnet/roslyn#12709]
While looking at [LambdaRewriter.cs](https://github.com/dotnet/roslyn/blob/master/src/Compilers/CSharp/Portable/Lowering/LambdaRewriter/LambdaRewriter.cs), I noticed that some lines are incorrectly indented. Specifically, it's the lines fixed by [this commit in my fork](https://github.com/svick/roslyn/commit/31faee02e1113e0e27d178d6ca10e0b0713020b0).

As far as I can tell, the incorrect changes are from [this merge](https://github.com/dotnet/roslyn/commit/e75ca21dffedb84f73a26b3038b6612236336a35#diff-e76253173c2d97f7e497cf28434637f5) from September 2015.

I haven't checked if other files have similar problems.

[The *Contributing Code* page in the repo wiki](https://github.com/dotnet/roslyn/wiki/Contributing-Code#code-formatter) says that Code Formatter is regularly run on the code in the repo. Presumably, running it would fix the indentation, so I guess it hasn't been run at least since September 2015?

My questions are: Should I just submit a PR with the commit above? Or should someone run the Code Formatter?

**Comment by svick** [comment]
Looks like that's not the only case: [this area of MethodCompiler.cs](https://github.com/dotnet/roslyn/blob/9617fa2/src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs#L591-L605) also has incorrect indentation, since [this October 2015 merge](https://github.com/dotnet/roslyn/commit/989eb92504eeb3c1abec32d5f2886b8552bfb989#diff-339de703849fd37b6e84a4aafbf1d779).

**Comment by hashhar** [comment]
I can run a script I have and let you know of such indentation issues. But I can't start before the weekend. I hope that's not much of an issue.

**Comment by hashhar** [comment]
Looks like the codeformatter has not been run in a very long time. I don't how should I proceed.

**Comment by svick** [comment]
@hashhar Yeah. I have submitted a PR with the results of running CodeFormatter (plus some manual followup changes): #13153.
